tests/e2e/java_test.go

```diff
@@ -32,7 +32,8 @@ var _ = Describe("odoJavaE2e", func() {
 		It("Should be able to deploy a git repo that contains a wildfly application", func() {
 
 			// Deploy the git repo / wildfly example
-			runCmd("odo create wildfly javaee-git-test --git " + warGitRepo)
+			cmpCreateLog := runCmd("odo create wildfly javaee-git-test --git " + warGitRepo + " -w")
```

Comment:
we should also have some java tests that don't have a wait flag set.

Reply:
https://github.com/redhat-developer/odo/pull/987/files#diff-3db9664e2b8df4cc9a81bb49c3b3303cR57